.github/workflows/renovate.yml

```diff
@@ -27,8 +25,14 @@ jobs:
           app-id: ${{ vars.BOT_APP_ID }}
           private-key: ${{ secrets.BOT_PRIVATE_KEY }}
       - env:
-          RENOVATE_TOKEN: ${{ secrets.RENOVATE_TOKEN || steps.generate-token.outputs.token }}
+          RENOVATE_TOKEN: ${{ steps.generate-token.outputs.token || secrets.RENOVATE_TOKEN }}
```

Comment:
The question here could be which will win when both `RENOVATE_TOKEN` and GitHub App Token are configured. Actually, I would suggest to have `RENOVATE_TOKEN` win since it is more explicitly. Maybe we should even skip GitHub App Token generation when `RENOVA_TOKEN` is specified.

Reply:
it does make sense.